From 3e7f916f12599df36baa4a8d472664e733e4d233 Mon Sep 17 00:00:00 2001
From: mydongistiny <jaysonedson@gmail.com>
Date: Tue, 22 May 2018 20:33:24 -0700
Subject: [PATCH 2/2] build: Change flex location to build-tools

Also remove use of host flex as it's not needed

Change-Id: Iab53689a0dcc5c5b16da34fd1d836ab7358f176d
Signed-off-by: mydongistiny <jaysonedson@gmail.com>

Conflicts:
	core/config.mk
---
 core/config.mk | 7 ++-----
 1 file changed, 2 insertions(+), 5 deletions(-)

diff --git a/core/config.mk b/core/config.mk
index a95aadb3a4..aa20e5b0a4 100644
--- a/core/config.mk
+++ b/core/config.mk
@@ -617,6 +617,7 @@ ACP := $(prebuilt_build_tools_bin)/acp
 CKATI := $(prebuilt_build_tools_bin)/ckati
 DEPMOD := $(HOST_OUT_EXECUTABLES)/depmod
 FILESLIST := $(SOONG_HOST_OUT_EXECUTABLES)/fileslist
+LEX := $(prebuilt_build_tools_bin)/flex
 MAKEPARALLEL := $(prebuilt_build_tools_bin)/makeparallel
 SOONG_JAVAC_WRAPPER := $(SOONG_HOST_OUT_EXECUTABLES)/soong_javac_wrapper
 SOONG_ZIP := $(SOONG_HOST_OUT_EXECUTABLES)/soong_zip
@@ -627,11 +628,7 @@ ZIPTIME := $(prebuilt_build_tools_bin)/ziptime
 
 # ---------------------------------------------------------------
 # Generic tools.
-ifeq ($(USE_HOST_LEX),true)
-  LEX := flex
-else
-  LEX := prebuilts/misc/$(BUILD_OS)-$(HOST_PREBUILT_ARCH)/flex/flex-2.5.39
-endif
+JACK := $(HOST_OUT_EXECUTABLES)/jack
 # The default PKGDATADIR built in the prebuilt bison is a relative path
 # prebuilts/build-tools/common/bison.
 # To run bison from elsewhere you need to set up enviromental variable
-- 
2.17.1

